apps/root/src/pages/aggregator/swap-container/components/quote-selection/index.tsx

```diff
@@ -165,44 +61,27 @@ const QuoteSelection = ({
   const { sorting } = useAggregatorSettingsState();
   const currentNetwork = useSelectedNetwork();
   const isPermit2Enabled = useIsPermit2Enabled(currentNetwork.chainId);
-  const [anchorEl, setAnchorEl] = React.useState<HTMLButtonElement | null>(null);
-  const dispatch = useAppDispatch();
-  const trackEvent = useTrackEvent();
+  const simulationsTimer = useSimulationTimer({
+    simulations: TOTAL_SIMULATIONS,
+    simulationInProgress: isPermit2Enabled && from?.address === PROTOCOL_TOKEN_ADDRESS && isLoading,
```

Comment:
shouldn't this just be isLoading?

Reply:
I didn't want the timer to start unless we actually need it to be triggered. With isLoading only, it would be running no matter if we simulate or not